### DISCUSSION TITLE
The weirdest bug I've ever seen! Standardnotes app Ipad would not load after making changes via the Desktop App

### DISCUSSION THREAD

**Discussion by BobWs** [standardnotes/forum#2827] [General]
This is probably the weirdest bug I've ever seen. 
(see screenshot)
![IMG_0103](https://user-images.githubusercontent.com/5485820/177727811-343fadda-5a93-4b75-9f4c-71fe6e71dc11.PNG)

If I make changes to my notes or folder/tags structure via the desktop app and then open standardnotes on the ipad I get a blank screen. 
The app starts up normally and after I log in with my fingerprint then I get a white screen. There is no way to fix this other than deleting the app and reinstalling it. 

I have had to do this at least five times in the past few days.
What have I already tried:
- restarting ipad
- disable fingerprint
- leave the app on white screen for a while to see if it recovers

The only thing that helps is reinstalling the app.

And why is it extremely strange, because this only happens on the ipad! On my iphone it just works. I have tested it on two different ipads with two different users and both have the same problem, namely a white screen after changing the notes via desktop.

I hope you will find a solution for this soon, because this problem occurs since I have updated to the latest version of standardnotes iOS.

**Comment by atmoio** [comment]
Which version of the app is this, and are you on the TestFlight?

**Comment by BobWs** [comment]
> Which version of the app is this, and are you on the TestFlight?

Version 3.23.8 (latest iOS version in the AppStore)
No not on TestFlight.

**Comment by BobWs** [comment]
Problem still present in version 3.30.25

**Comment by jacobgkau** [comment]
I'm having what may be the same issue on an Android tablet, a Samsung Galaxy Tab S2 8.0 running LineageOS. I was struggling to figure out why the app just completely stopped working the other day, but now that I think about it, I switched my desktop app to no longer use the themed menu bar a few days ago too, so that could be correlated.

Since I'm on a paid plan, I emailed the help@ address this weekend, but haven't heard back yet. Unlike your iPad report, uninstalling/reinstalling doesn't fix it on my Android tablet (nor does completely resetting/clearing cache & storage of the app.)
